test/nbrowser/HomeIntro.ts

```diff
@@ -129,6 +130,22 @@ describe('HomeIntro', function() {
     }
   }
 
+  async function testMetaTags() {
+    const expectedTitle = 'Grist, the evolution of spreadsheet';
+    assert.equal(await driver.find('meta[name="twitter:title"]').getAttribute('content'), expectedTitle);
+    assert.equal(await driver.find('meta[property="og:title"]').getAttribute('content'), expectedTitle);
+
+    const expectedDescription = 'A modern, open source spreadsheet that goes beyond the grid';
```

Comment:
I don't know if that description fits well in that case:

![image](https://github.com/user-attachments/assets/50a7e03f-6c4d-4c2f-9cc8-5d64b0031f63)

Still that seems OK to me anyway, but alternative proposals for the default document description are welcome :)